src/app/modules/rup/components/ejecucion/hudsBusqueda.component.ts

```diff
@@ -351,7 +351,7 @@ export class HudsBusquedaComponent implements AfterContentInit {
             this.servicioPrestacion.getByPacienteSolicitud(this.paciente.id).subscribe((solicitudes) => {
 
                 solicitudes.forEach(solicitud => {
-                    const prestacion = this.prestacionesTotales.find(p => solicitud.idPrestacion === p.solicitud.prestacionOrigen);
+                    const prestacion = this.prestacionesTotales.find(p => solicitud.idPrestacion === p.solicitud.prestacionOrigen && solicitud.concepto.conceptId === p.solicitud.tipoPrestacion.conceptId);
```

Comment:
https://www.loom.com/share/e3f471f067d6451f9c1cef3ddfe147ee
Al crear más de una solicitud dentro de una prestación, cuando se "cruzan" las prestaciones con las solicitudes para armar el listado de historia clínica, `this.prestacionesTotales` tendra tantos objetos con id de la prestación madre, como solicitudes contenidas. Por eso es preciso agregar el filtro de concepto de prestación solicitada, de otra manera siempre retornara los datos de la primer solicitud que referencie al id de la prestación.